tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/SignificantIndentationSuite.scala

```diff
@@ -54,6 +54,54 @@ class SignificantIndentationSuite extends BaseDottySuite {
     )
   }
 
+  test("anonymous-class") {
```

Comment:
you are testing anonymous classes. did it work, without this change, with regular
```
class A:
  ...
```
and does it work with early defs (that is, should line 4733 be changed as well):
```
class A extends { def ... }:
  ...
```

Reply:
The first one worked previously, only `new:` was not handled.

As for `early defs` they are no longer allowed in Dotty, so this is safe.